zone/raids.cpp

```diff
@@ -1595,8 +1613,9 @@ void Raid::VerifyRaid()
 		}
 		else{
 			Client *c = entity_list.GetClientByName(members[x].membername);
-			Bot* b;
-			if(c){
+			Bot* b = entity_list.GetBotByBotName(members[x].membername);
```

Comment:
Did you mean to revert this? Can probably set default as `nullptr`.

Reply:
same